astropy/units/tests/test_physical.py

```diff
@@ -477,3 +478,34 @@ def teardown_method(self):
                 f"the physical type for {unit}, which was added for"
                 f"testing, was not deleted."
             )
+
+
+@pytest.mark.parametrize(
+    "method, expected",
+    [
+        ("title", 'Length'),
+        ("isalpha", True),
+        ("isnumeric", False),
+        ("upper", 'LENGTH')
+    ],
+)
```

Comment:
My preference is also for ``black`` but https://docs.astropy.org/en/stable/development/codeguide.html?highlight=black#coding-style-conventions.

Reply:
I did change the style a bit in response, but any of the maintainers should feel free to suggest and accept stylistic changes.

Reply:
> The perception might change in the future, so who knows.

I've been itching to do ``>>> black astropy``  and git push for years now. 🤣

Reply:
No, please, no!!